action.php

```diff
@@ -420,16 +478,16 @@ protected function generatePDF($cachefile) {
         $no_pages = count($this->list);
         foreach($this->list as $page) {
             $counter++;
-            $filename = wikiFN($page, $REV);
-
-            if(!file_exists($filename)) {
-                continue;
-            }
 
             // set global pageid to the rendered page
             $ID = $page;
 
-            $pagehtml = p_cached_output($filename, 'dw2pdf', $page);
+            //$pagehtml = p_cached_output($filename, 'dw2pdf', $page);
+            $pagehtml = $this->p_wiki_dw2pdf($ID, $rev, $date_at);
```

Comment:
` $ID = $page;` above this function is duplicated in the new function. The original one is not needed I guess (or page_depend_replacements() or $mpdf->WriteHTML()? is still secretly using it somewhere in certain manner...)

Reply:
No, that should be fine. Thank you for the catch 🙂